commands/functions-shell.js

```diff
@@ -17,6 +17,7 @@ var scopes = require('../lib/scopes');
 module.exports = new Command('experimental:functions:shell')
   .description('launch full Node shell with emulated functions')
   .option('-p, --port <port>', 'the port on which to emulate functions (default: 5000)', 5000)
+  .option('-t, --timeout <timeout>', 'how long the function can be idle before being terminated, in seconds. The default is 60 seconds.')
```

Comment:
I'd simplify this to: "timeout for all emulated functions in seconds (default: 60)". Since timeout is the GCF terminology, and our customers don't need to know the intricacies of idle pruning

Reply:
also should add flag to firebase-serve as well. But make sure the description is appended with [for functions only]